lib/ecto/adapters/mysql/connection.ex

```diff
@@ -1,6 +1,18 @@
-if Code.ensure_loaded?(Mariaex.Connection) do
+# Does not compile without specifying this 
+alias Mysqlex, as: MysqlDriver
```

Comment:
Getting a struct not defined error on compile if this line is not included, which foregoes the point of conditionally including drivers.